### PR TITLE
webapp: Adding support for MinTls, AlwaysOn, FtpsState properties for an app & slot

### LINKED ISSUE #10851: Set-AzWebapp should support AlwaysOn
## Description of the new feature
Support set alwaysOn similar to https://docs.microsoft.com/en-us/cli/azure/webapp/config?view=azure-cli-latest#az-webapp-config-set

### PR DESCRIPTION
**Pull request by panchagnula**
Fixes #10851, #10162, #6130

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Websites/Websites/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Set-AzWebapp and Set-AzWebappSlot supports AlwaysOn, MinTls and FtpsState properties
+* Fixing issue where etting HttpsOnly along with changing AppservicePlan at the same time using the single Set-AzWebApp Command, was resetting HttpsOnly to default value
```
Comment (isra-fel) [inline src/Websites/Websites/ChangeLog.md]:
```suggestion
* Fixing issue where editing HttpsOnly along with changing AppServicePlan at the same time using the single Set-AzWebApp Command, will reset HttpsOnly to default value
```

Comment (panchagnula) [reply src/Websites/Websites/ChangeLog.md]: oops will fix the typo as well

File: src/Websites/Websites/Cmdlets/DeploymentSlots/SetAzureWebAppSlot.cs [hunk 2]
```diff
@@ -158,7 +177,8 @@ public override void ExecuteCmdlet()
                             Use32BitWorkerProcess =
                                 parameters.Contains("Use32BitWorkerProcess") ? (bool?)Use32BitWorkerProcess : null,
                             AutoSwapSlotName = parameters.Contains("AutoSwapSlotName") ? AutoSwapSlotName : null,
-                            NumberOfWorkers = parameters.Contains("NumberOfWorkers") ? NumberOfWorkers : WebApp.SiteConfig.NumberOfWorkers
+                            NumberOfWorkers = parameters.Contains("NumberOfWorkers") ? NumberOfWorkers : WebApp.SiteConfig.NumberOfWorkers,
+                            AlwaysOn = parameters.Contains("AlwaysOn") ? (bool)AlwaysOn : false
```
[the remark this answers is not in the document]
Comment (panchagnula) [reply src/Websites/Websites/Cmdlets/DeploymentSlots/SetAzureWebAppSlot.cs]: @isra-fel I remember going through this exercise last year see this change https://github.com/panchagnula/azure-powershell/commit/6950697b0a1fb290b9da529c1a8a368000e63592#diff-d9f9b573dfd0ebf3ccdec4ce4a59336a but after that there was some reason this was decided to be changed to bool again hence the reason I followed the current pattern. Let me know if this is still needed. Thanks!

[the remark this answers is not in the document]
Comment (isra-fel) [reply src/Websites/Websites/Cmdlets/DeploymentSlots/SetAzureWebAppSlot.cs]: I see, your API was designed to have three state: true, false and null, so it's mapped to an optional boolean parameter. Makes sense

**Pull request merged**